src/components/Synthetics/TableMarketFilter/MarketFilterLongShort.tsx

```diff
@@ -0,0 +1,60 @@
+import { useLingui } from "@lingui/react";
+import { defineMessage } from "@lingui/macro";
+import mapValues from "lodash/mapValues";
+import { useCallback, useMemo } from "react";
+import type { MessageDescriptor } from "@lingui/core";
+
+import Tab from "components/Tab/Tab";
+import { MarketFilterBase, MarketFilterBaseProps } from "./MarketFilterBase";
+
+import "./MarketFilterLongShort.scss";
+
+type MarketFilterProps = Omit<MarketFilterBaseProps, "beforeContent" | "extraIsActive"> & {
+  valueIsLong: boolean | undefined;
+  onChangeIsLong: (value: boolean | undefined) => void;
+};
+
+type State = "all" | "long" | "short";
```

Comment:
not a biggy but `Option` instead of `State` should make more sense

Reply:
done